### DISCUSSION TITLE
Other Update possibilities

### DISCUSSION THREAD

**Discussion by ISeeTWizard** [firefly-iii/firefly-iii#4744] [Firefly III]
Hi
I just wanted to know if there is another update possibility apart always deleting and recreating the site?

just with as example composer update or so? (Maybe a json file needs to be changed or so)
Or simply overwriting the files and than lauching an db update procedure?

for now doing composer update several componants are updated but not firefly in itself (still on 5.5.8 and as far as I see 5.5.10 is out)

thanks

**Comment by ISeeTWizard** [comment]
Forget it
Found a script for it and I gonna try out this one…

**Comment by JC5** [comment]
If the Docker upgrade doesn't work, what's the tag you're using?

An upgrade through Git is also possible. Clone the repo, do `git pull` when there's a new version.
